xilem/src/view/split.rs

```diff
@@ -0,0 +1,276 @@
+    ///
+    /// If this is `false` (the default), the bar will be drawn as two parallel lines.
+    pub fn solid_bar(mut self, solid: bool) -> Self {
+        self.solid_bar = solid;
+        self
+    }
+}
+
+const CHILD1_VIEW_ID: ViewId = ViewId::new(0);
+const CHILD2_VIEW_ID: ViewId = ViewId::new(0);
+
+impl<ChildA, ChildB, State, Action> ViewMarker for Split<ChildA, ChildB, State, Action> {}
+impl<ChildA, ChildB, State, Action> View<State, Action, ViewCtx>
+    for Split<ChildA, ChildB, State, Action>
+where
+    State: 'static,
+    Action: 'static,
+    ChildA: WidgetView<State, Action>,
+    ChildB: WidgetView<State, Action>,
+{
+    type Element = Pod<widgets::Split<ChildA::Widget, ChildB::Widget>>;
+
+    type ViewState = (ChildA::ViewState, ChildB::ViewState);
+
+    fn build(&self, ctx: &mut ViewCtx) -> (Self::Element, Self::ViewState) {
+        let (child1, child1_state) = ctx.with_id(CHILD1_VIEW_ID, |ctx| self.child1.build(ctx));
+        let (child2, child2_state) = ctx.with_id(CHILD2_VIEW_ID, |ctx| self.child2.build(ctx));
+
+        let widget_pod = ctx.new_pod(
+            widgets::Split::new_pod(child1.into_widget_pod(), child2.into_widget_pod())
+                .split_axis(self.split_axis)
+                .split_point(self.split_point)
+                .min_size(self.min_size.0, self.min_size.1)
+                .bar_size(self.bar_size)
+                .min_bar_area(self.min_bar_area)
+                .draggable(self.draggable)
+                .solid_bar(self.solid_bar),
+        );
+
+        (widget_pod, (child1_state, child2_state))
+    }
+
+    fn rebuild(
+        &self,
+        prev: &Self,
+        view_state: &mut Self::ViewState,
+        ctx: &mut ViewCtx,
+        mut element: xilem_core::Mut<'_, Self::Element>,
+    ) {
+        if prev.split_axis != self.split_axis {
+            widgets::Split::set_split_axis(&mut element, self.split_axis);
+        }
+
+        if prev.split_point != self.split_point {
+            widgets::Split::set_split_point(&mut element, self.split_point);
+        }
+
+        if prev.min_size != self.min_size {
+            widgets::Split::set_min_size(&mut element, self.min_size.0, self.min_size.1);
+        }
+
+        if prev.bar_size != self.bar_size {
+            widgets::Split::set_bar_size(&mut element, self.bar_size);
+        }
+
+        if prev.min_bar_area != self.min_bar_area {
+            widgets::Split::set_min_bar_area(&mut element, self.min_bar_area);
+        }
+
+        if prev.draggable != self.draggable {
+            widgets::Split::set_draggable(&mut element, self.draggable);
+        }
+
+        if prev.solid_bar != self.solid_bar {
+            widgets::Split::set_bar_solid(&mut element, self.solid_bar);
+        }
+
+        let child1_element = widgets::Split::child1_mut(&mut element);
+        self.child1
+            .rebuild(&prev.child1, &mut view_state.0, ctx, child1_element);
```

Comment:
```suggestion
        ctx.with_id(CHILD1_VIEW_ID, |ctx| {
            self.child1
                .rebuild(&prev.child1, &mut view_state.0, ctx, child1_element)
        });
```

Same for the other child.

As explained below as well, tree-structure might change in this function (within children), so we need to track this via the id path, so that messages can be routed correctly in `View::message`.